content/04.results.md

```diff
@@ -0,0 +1,16 @@
+# Results
+
+1. Figure 1 - Visual to show that alignment works along intra year variation (Procrustes Validation)
+  1. Umap panel for an individual year to show models are separated
```

Comment:
I would start with the simplest possible way that you can show things, and only bring in more complexity like UMAP when necessary. I'm guessing that you would be able to show this with just PCA or similar, unless I'm not thinking correctly.